docs/canfail.md

```diff
@@ -33,6 +33,9 @@ Code | Rewrite
 `uio.retryOrElseEither(s, f)` | `uio`*
 `uio.tapBoth(f, g)` | `uio.tap(g)`
 `uio.tapError(f)` | `uio`
+`ZIO.partitionM(in)(f)` | `ZIO.foreach(in)(f)`*
```

Comment:
We should generalize the footnote so it covers these cases.

Reply:
What do you mean ?

Reply:
At the bottom of that page there is a footnote that reads:

```scala
* `either`, `option`, `orElseEither`, and `retryOrElseEither` wrap their results in `Some` or `Right` so after rewriting, code calling these methods can be simplified to accept an `A` rather than an `Option[A]` or `Either[E, A]`.
```

We should change the wording slightly so that it covers the new cases we are adding. Something along the lines of "These methods wrap their results in an `Option` or an `Either`, so after rewriting, code calling these methods can be simplified to accept a value instead of an `Option` or an `Either`."

Reply:
Oooooh Yeah I will update it